shared/ui-components/src/authentication/UserContext.tsx

```diff
@@ -140,11 +140,13 @@ export const UserProvider: FunctionComponent = ({ children }) => {
     accessToken: state.accessToken,
     initialStateLoaded: state.initialStateLoaded,
     login: async (email, password) => {
-      dispatch(signOut())
```

Comment:
Unless it's causing problems, this should still be here I think. The goal is a "belt and suspenders" check to clear out any existing user information before we login the user.